packages/react-dom/src/__tests__/ReactDOMComponent-test.js

```diff
@@ -14,20 +14,31 @@ describe('ReactDOMComponent', () => {
   var ReactTestUtils;
   var ReactDOM;
   var ReactDOMServer;
-  var inputValueTracking;
+
+  var getInputValueTracker;
+  var getTextAreaValueTracker;
 
   function normalizeCodeLocInfo(str) {
     return str && str.replace(/\(at .+?:\d+\)/g, '(at **)');
   }
 
   beforeEach(() => {
     jest.resetModules();
+
+    getInputValueTracker = Object.getOwnPropertyDescriptor(
```

Comment:
I don't understand what this does in the context of this test.

As @jquense explained above, the original purpose of the test is to verify our custom tracker object "sees" the original value.

However this notion of "tracker" is not the same thing that you get from the HTMLInputElement's prototype. I think you got confused by naming used in some other test.

The thing you read from HTMLInputElement prototype *is just the jsdom's value accessor itself*. So there is no need to test it at all. You're just testing that jsdom works. (But we already know it works as it has its own test suite :-)